R/plotHeatmap.R

```diff
@@ -19,6 +19,8 @@
 #' @param colour_columns_by A list of values specifying how the columns should be annotated with colours.
 #' Each entry of the list can be any acceptable input to the \code{by} argument in \code{?\link{retrieveCellInfo}}.
 #' A character vector can also be supplied and will be treated as a list of strings.
+#' @param column_annotation_colors Passed to \code{\link[pheatmap]{pheatmap}} 
```

Comment:
Almost there. Just mention the expected type, e.g., "A named list specifying the color scales to be used for any of the column annotations. Names should be set to (character) entries of `colour_columns_by`. If a color scale is not specified for a particular annotation, a default color scale is chosen."

Reply:
Good call, was originally being lazy and just passing the buck to pheatmap but this is friendlier behaviour